### PR TITLE
DAOS-6046 test: Fix unit test coverage build

### PR DESCRIPTION
**Pull request by sylviachanoiyee**
Skip-bullseye: false
Skip-unit-test-memcheck: true
Skip-checkpatch: true
Skip-build-leap15-icc: true
Skip-coverity-test: true
Skip-func-hw-test-small: true
Skip-func-hw-test-medium: true
Skip-func-hw-test-large: true
Skip-func-test-leap15: true
Skip-build-leap15-rpm: true
Skip-func-test: true

### CODE REVIEW

**Review by jolivier23 — DISMISSED** [review]

**Review by JohnMalmberg — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -756,8 +756,7 @@ pipeline {
                             filename 'Dockerfile.centos.7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " +
-                                '$BUILDARGS_QB_TRUE' +
+                            additionalBuildArgs dockerBuildArgs(qb: quickbuild()) +
```
Comment (brianjmurrell) [inline Jenkinsfile:759]: We shouldn't eliminate the `-t ...` argument.
```suggestion
                            additionalBuildArgs "-t  ${sanitized_JOB_NAME}-centos7 " +
                                                dockerBuildArgs(qb: quickbuild()) +
```

It's true that `-t ...` is not used consistently throughout this file.  But it is used probably a majority of the time and is a bug where it's not used.  You could add those in this PR if you wanted to.  :-)  Or not, but at least not remove it from here.

File: Jenkinsfile [hunk 2]
```diff
@@ -1419,8 +1418,7 @@ pipeline {
...
```
Comment (brianjmurrell) [inline Jenkinsfile:1421]: Ditto.

It's probably worth mentioning that it's probable that this `-t ...` argument could be added to [dockerBuildArgs()](https://github.com/daos-stack/pipeline-lib/blob/master/vars/dockerBuildArgs.groovy), but that's a separate effort than what you are fixing here.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (sylviachanoiyee) [reply Jenkinsfile:759]: ok

**Comment by ashleypittman** [comment]
As per discussion on Skype can you try enabling this by default, and setting the pass thresholds to just below the current number so that it's on, and passing in regular builds but will fail if there's a regression in coverage.  The most recent two builds have passed but not actually run with the test enabled.

**Review by JohnMalmberg — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**